streamlit_parameters/demos/parameters.py

```diff
@@ -214,7 +214,7 @@ def main():
     streamlit.write("## Debugging")
 
     streamlit.write("#### Query String")
-    query_string: typing.Dict[str, str] = streamlit.experimental_get_query_params()
+    query_string: typing.Dict[str, str] = streamlit.query_params.get_all()
```

Comment:
Perhaps the api changed (only works for get_all on a single key now).

This misses repeated keys, but that's fine since we're only using it for debugging.